crates/core/tedge_mapper/src/core/converter.rs

```diff
@@ -67,6 +68,25 @@ pub trait Converter: Send + Sync {
     fn sync_messages(&mut self) -> Vec<Message> {
         vec![]
     }
+
+    fn try_process_operation_update_message(
+        &mut self,
+        _input: &DiscoverOp,
+    ) -> Result<Option<Message>, Self::Error> {
+        Ok(None)
+    }
+
+    fn process_operation_update_message(&mut self, message: DiscoverOp) -> Message {
+        let message_or_err = self.try_process_operation_update_message(&message);
+        match message_or_err {
+            Ok(Some(msg)) => msg,
+            Ok(None) => Message::new(
+                &self.get_mapper_config().errors_topic,
+                "No operation update required",
+            ),
```

Comment:
I would do nothing in that case.

To fix that, you need to fix the function signature, returning an `Option<Message>`.